content/applications/sales/crm/performance.rst

```diff
@@ -8,3 +8,4 @@ Analyze performance
    :titlesonly:
 
    performance/win_loss
+   performance/google_spreadsheets
```

Comment:
It's not related to auth neither.
And the rare use cases where it could be useful are in CRM reporting. So, I thought people would have more chances to see it there, rather than the Generc / Auth section.